File: app/src/main/java/net/todd/mavelheroes/data/MarvelCharacter.java
package net.todd.mavelheroes.data;

public class MarvelCharacter {
    private String description;
    private String name;
    private String id;
    private CharacterThumbnail thumbnail;

    public String getName() {
        return name;
    }

    public String getDescription() {
        return description;
    }

    public String getImagePath() {
        return thumbnail.getImagePath();
    }

    public String getId() {
        return id;
    }
}

File: app/src/main/java/net/todd/mavelheroes/net/todd/mavelheroes/data/MarvelCharacter.kt
package net.todd.mavelheroes.net.todd.mavelheroes.data

class MarvelCharacter {
    var id: String? = null
    var name: String? = null
    var description: String? = null
    var thumbnail: CharacterThumbnail? = null

    constructor() {
    }

    constructor(id: String, name: String, description: String, thumbnail: CharacterThumbnail) {
        this.id = id
        this.name = name
        this.description = description
        this.thumbnail = thumbnail
    }

    val imagePath: String
        get() = thumbnail!!.imagePath
}
